sphinx/writers/latex.py

```diff
@@ -361,6 +364,52 @@ def get_colspec(self):
         else:
             return '{|' + ('l|' * self.colcount) + '}\n'
 
+    def add_cell(self, height, width):
+        self.cell_id += 1
+        for col in range(width):
+            for row in range(height):
+                assert self.cells[(self.row + row, self.col + col)] == 0
+                self.cells[(self.row + row, self.col + col)] = self.cell_id
+
+    def cell(self, row=None, col=None):
+        try:
+            if row is None:
+                row = self.row
+            if col is None:
+                col = self.col
+            return TableCell(self, row, col)
+        except IndexError:
+            return None
+
+
+class TableCell(object):
+    def __init__(self, table, row, col):
+        if table.cells[(row, col)] == 0:
+            raise IndexError
+
+        self.table = table
+        self.cell_id = table.cells[(row, col)]
+        for n in range(row + 1):
+            if table.cells[(row - n, col)] == self.cell_id:
+                self.row = row - n
```

Comment:
actually, now that I understand better that cells are rectangles doing a "pavement" of the table, perhaps something like:
```
self.row = row - 1
while self.row >= 0 and table.cells[(self.row, col)] == self.cell_id:
    self.row -= 1
self.row += 1
```